ngirimana/murikahomeltd
# #173241998 Add signup

> #### What does this PR do?
* add signup form
> #### Description of Task to be completed
- Add spinner
- Add heading
- add links
> #### How should this be manually tested?
* clone repository
* run ``` cd murika-front-end``` in terminal
* checkout  **ft-signup-api-integration-173241998**  branch by running ```git checkout ft-signup-api-integration-173241998``` in terminal
* use 'npm install' to install all necessary dependencies
* Run 'npm start' to run the server 
* And click the signup link
> #### Any background context you want to provide?
* you should install react  in your PC 
> #### What are the relevant pivotal tracker stories?
* [#173241998](https://www.pivotaltracker.com/story/show/173241998)
> #### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38111965/84142163-9b6fed80-aa54-11ea-8f35-427eb26d31f4.png)
